### DISCUSSION TITLE
Incremental filtering support

### DISCUSSION THREAD

**Discussion by data-man** [dail8859/NotepadNext#279] [General]
SciTE has this feature:

![ksnip_20220510-083755](https://user-images.githubusercontent.com/1754269/167537912-58d51ad6-e79b-47b9-9185-fe1f082a1d31.png)

![ksnip_20220510-084123](https://user-images.githubusercontent.com/1754269/167538200-7bdb24c2-80e1-40a2-8331-9e9bc90deab9.png)

The nice project, thank you!

**Comment by dail8859** [comment]
I'm not against this feature but I'm converting this over to a discussion for anyone interested in it.
